docs/index.md

```diff
@@ -1,94 +1,217 @@
-# Welcome to LearnD3D11
+# Welcome to Learning Direct3D11
 
-!!! note
-    This thing is a work in progress.
+## Introduction
 
-## Sub Chapter 1
+Goal of this thing
 
-=== "Windows"
+### DX API Overview
 
-    - Instruction for windows
+DirectX consists of various components such as
 
-    === "SubTab1"
+- `Direct3D11` elaborate...
 
-        SubTab1Content
+- `Direct2D1` elaborate...
 
+- `D3DCompiler` elaborate...
 
-    === "SubTab2"
+- `XAudio2` elaborate...
 
-        Sub2Content
+- `XACT` elaborate...
 
-    === "SubTab3"
+- `DirectWrite` elaborate...
 
-        SubTab3Content
+- `XInput` elaborate...
 
-        !!! note
+but we are going to focus on mainly `Direct3D11`
 
-        That one costs money
+### Direct3D11 Usage
 
-    Clone repo
-    ```bash
-    some bash instructions for reference
-    ```
+Direct3D11 comes with quite a bit of classes and constructs to make use of the API.
 
-=== "Ubuntu"
+- `ID3D11Device` elaborate...
 
-    long list of instructions
-    and more instruction
+- `ID3D11DeviceContext` elaborate...
 
-=== "Fedora"
+- `ID3D11Texture` elaborate...
 
-    === "SubTab1"
+- `ID3D11ShaderResourceView` elaborate...
 
-    Again instructions
-    and more stuff here
+- `ID3D11UnorderedAccessViw` elaborate...
 
-    === "SubTab2"
+- `ID3D11PixelShader` elaborate...
 
-    SubTab2Content
+- `ID3D11VertexShader` elaborate...
 
-=== "ArchLinux"
+- `ID3D11InputLayout` elaborate...
 
-    !!! info
+- `ID3D11RasterizerState` elaborate... asd
 
-        Some information inside this tab
+### Libraries & Project Layout
 
-    ```cs
-    class SomeCSharpCodeHere
+For the LearnD3D11 tutorial series we will be using the following third party libraries
+
+- [GLFW](https://www.glfw.org) It provides the window to render our things in, as well as input handling via mouse and keyboard. Yes there are also alternatives like `SDL2` for instance. It serves the same purpose, we just decided to go with `GLFW`. You might also wonder about "I want to do it myself, from scratch" then you really could do it yourself; but its the year 2022 by the time we are writing this thing and the problem of creating a window and handling input has been solved many times already, why reinvent the wheel. We have a link [here](DIYWinApi.md) to show how you could do it).
```

Comment:
For readability of the raw Markdown, please split this into multiple lines. Otherwise one has to scroll horizontally for quite a while.

Reply:
You are welcome to suggest explicitly already

Reply:
I'm on my phone currently, but probably manually adding linebreaks at ~80/100 characters should be sufficient. The content itself seems good to me.